document/core/exec/instructions.rst

```diff
@@ -3146,20 +3149,28 @@ on the top of the stack.
    \CATCHadm\{a'^?~\instr'^\ast\}^\ast~\XT[\val^n~(\THROWadm~a)]~\END  \\
    && (\iff a_1^? \neq \epsilon \land a_1^? \neq a) \\
    S;~\CATCHadm\{a_1^?~\instr^\ast\}\{a'^?~\instr'^\ast\}^\ast~\XT[\val^n~(\THROWadm~a)]~\END &\stepto&
-   S;~\CAUGHTadm\{a~\val^n\}~(\val^n)?~\instr^\ast~\END \\
+   S;~\CAUGHTadm\{a~\val^n\}~(\val^n)^?~\instr^\ast~\END \\
    && (\iff~(a_1^? = \epsilon \lor a_1^? = a)~\land\\
    && \ S.\STAGS[a].\TAGITYPE = [t^n]\to[]) \\
    \LABEL_n\{\}~\XB^l[\DELEGATEadm\{l\}~\XT[\val^n~(\THROWadm~a)]~\END]~\END &\stepto&
    \XT[\val^n~(\THROWadm~a)]  \\
    \end{array}
 
 .. note::
-   The reduction step activating a catch clause |CAUGHTadm| is the only one that does not preserve the throw context.
-   While a |THROWadm| propagates through the stack, the throw context |XT| is collected
-   until the exception is caught inside a |CAUGHTadm| instruction, at which point it is discarded.
+   The reduction step entering a catch clause |CAUGHTadm| is the only one that does not preserve the throw context.
 
 .. todo::
-   Add explainer note for the reduction of |DELEGATEadm|, when PR #221 is settled.
+   Explain why or what's the meaning of apparently preserving the throw context in the other rules.
+
+
+.. _exec-caughtadm-enter:
+
+Entering :math:`\instr^\ast` with catch clause holding a caught exception :math:`\{a~\val^n\}`
```

Comment:
See comment above.

Reply:
This sounds like the answer to my question [above](https://github.com/WebAssembly/exception-handling/pull/226/files#r1101318259), is to remove both sections `_exec-caughtadm-enter` and `_exec-caughtadm`, and merge their steps into the `exec-throwadm` prose directly?